src/options.ts

```diff
@@ -23,11 +24,23 @@ const jitiDefaults: JITIOptions = {
   esmResolve: _EnvESMResolve || false,
   cacheVersion: "7",
   legacy: lt(process.version || "0.0.0", "14.0.0"),
-  extensions: [".js", ".mjs", ".cjs", ".ts", ".mts", ".cts", ".json"],
+  extensions: [
+    ".js",
+    ".mjs",
+    ".cjs",
+    ".ts",
+    ".mts",
+    ".cts",
+    ".json",
+    ...(_EnvJsx ? [".jsx", ".tsx"] : []),
+  ],
   alias: _EnvAlias,
   nativeModules: _EnvNative || [],
   transformModules: _EnvTransform || [],
   experimentalBun: _ExpBun === undefined ? !!process.versions.bun : !!_ExpBun,
+  transformOptions: {
```

Comment:
This could be overwritten by user config. Should we use `defu` to merge the `jitiDefaults` and `userOptions`.

Reply:
For this PR, we can simply spread user options. We can consider defu in later PR 👍🏼